src/view/TabOverflowHook.tsx

```diff
@@ -33,7 +33,7 @@ export const useTabOverflow = (
 
     React.useEffect(() => {
         const instance = selfRef.current!;
-        instance.addEventListener('wheel', onWheel);
+        instance.addEventListener('wheel', onWheel, { passive: true });
```

Comment:
this needs to be passive: false so that the default mouse wheel event can be prevented (the reason we are adding the wheel handler here).
Explicitly setting passive: false does prevent the message in chrome

Reply:
If you make that change then I can merge your PR

Reply:
Thanks for getting back! I've changed it passive: false.